packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -711,6 +728,15 @@ export class AppManagementClient implements DeveloperPlatformClient {
       variables,
     )
   }
+
+  private async organizationBetaFlags(_organizationId: string, allBetaFlags: string[]): Promise<{[key: string]: boolean}> {
+    // For now, stub everything as false
```

Comment:
I'm not sure why we're intending to ship something like this -- is there a reason why the dependency (org flags, the 2nd point from the PR description) is not included _first_?